appserver/admingui/common/src/main/resources/org/glassfish/common/admingui/Strings.properties

```diff
@@ -15,6 +15,11 @@
 # SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 #
 
+## URLs to static resources.
```

Comment:
I don't like it as it is rather a hack than a fix. But while there is still the wrong "pragma" in AdminUI working with caches this way ...
Perhaps we should set something like this instead: https://stackoverflow.com/questions/6208715/pragma-no-cache-and-javascript

Reply:
Yes, it makes sense. But it's not the point of this PR and could be done later, what do you think? Once we resolve it in a better way, we can remove the additional query parameter and we will need to do it only in a single place.

Reply:
That's why I approved it. Generally we should do fixes correctly and just once, not hack and postpone to forever. But with the code in current state it might be more difficult, so lets do this step.